quipucords/scanner/network/runner/roles/ip/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: internal_host_started_processing_role
+  set_fact:
+    internal_host_started_processing_role: "ip address show"
+
+- name: gather 'ip address show' fact
+  raw: /sbin/ip address show
+  register: internal_ip_address_show_cmd
+  ignore_errors: yes
+  become: '{{user_has_sudo}}'
+  when: 'internal_have_ip'
+
+- name: extract result value for 'ip address show'
+  set_fact:
+    # NOTE: We want to process the related output in two different ways, but the current
+    # quipucords Processor class behavior requires us to include is as two facts.
+    ip_address_show_ipv4: "{{ internal_ip_address_show_cmd | default(None) }}"
+    ip_address_show_mac: "{{ internal_ip_address_show_cmd | default(None) }}"
```

Comment:
what if we go NUTS and save only one raw fact, skip the processor, and do that processing on fingerprinting instead? please PUSH BACK XD

Reply:
that way the output in details report will truly be a raw fact heh

Reply:
Hmm… That's not a bad idea. I might be eager to make a more aggressive change like that _if_ we were dropping the old `ifconfig` code, _but_ since we are keeping it, I want to minimize the blast radius here, and I don't want to touch any of the `ifconfig` code unless strictly necessary.

This assumes we _do_ want to keep the old `ifconfig`, though, which I assume we do since it probably still works on older versions of RHEL. Lots of qualifiers here because I'm afraid of breaking support for old systems as part of a bug fix for newer systems. 🙂